st2common/tests/unit/test_rbac_resolvers_inquiry.py

```diff
@@ -0,0 +1,272 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
Also need to update "rbac_for_supported_api_endpoints" tests and add additional RBAC tests for inquiry API endpoints.

Reply:
I'm assuming you're talking about something like `st2api/tests/unit/controllers/exp/test_inquiries_rbac.py`, which I have had as an untracked file for about a week now. :smile: Will get on that as soon as I'm done with the existing comments